exercises/clock/.meta/gen.go

```diff
@@ -44,6 +45,10 @@ type testGroup struct {
 	} `property:"equal"`
 }
 
+func (groups testGroup) GroupShortName() string {
+	return strings.ToLower(strings.Fields(groups.Description)[0])
```

Comment:
I have acknowledge this weakness in the commit message and noted that I find it better than hard-coding the names `addTests` and `subtractTests`. We will deal with it if it becomes a problem later on.